src/site/shared/containers/Header/Right/UtilitiesDropdown.tsx

```diff
@@ -0,0 +1,55 @@
+import {connect} from "react-redux";
+
+import {SharedAppState} from "shared/state";
+import {HeaderMenus, HeaderPopups} from "shared/state/Header/state";
+import {OpenHeaderMenu, OpenHeaderPopup, CloseHeaderMenus} from "shared/state/Header/actions";
+
+import {Dropdown} from "./Dropdown";
+
+type Utility = {
+    popupName: HeaderPopups;
+    img: string;
+    text: string;
+}
+
+type OwnProps = {}
+type StateProps = {
+    curMenu: HeaderMenus;
+}
+type DispatchProps = {
+    openMenu: (menu: HeaderMenus) => void;
+    openPopup: (popup: HeaderPopups) => void;
+    closeMenus: () => void;
+}
+type ExtraUtilitiesProps = {
+    extraUtilities: Utility[];
+}
+
+type Props = StateProps & DispatchProps & ExtraUtilitiesProps & OwnProps;
+const _UtilitiesDropdown = ({ curMenu, openMenu, openPopup, closeMenus, extraUtilities }: Props) => (
+    <Dropdown open={(curMenu === "tools")}
+              onClick={() => openMenu("tools")}
+              onClose={() => closeMenus()}
+              btnInfo={{title: "Utilities", src: "img/icons/tools.svg"}}>
+        {extraUtilities.map(tool => (
+            <div key={tool.popupName} onClick={() => { closeMenus(); openPopup(tool.popupName); }}>
```

Comment:
Only need `key` for dynamic elements that are created in loops

Reply:
Removing the key attribute results in an error in the chrome developer console:

index.js:1 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `_UtilitiesDropdown`. See https://reactjs.org/link/warning-keys for more information.
    at div
    at _UtilitiesDropdown (http://localhost:3000/static/js/main.chunk.js:14014:3)
    at ConnectFunction (http://localhost:3000/static/js/vendors~main.chunk.js:49834:75)
    at div
    at HeaderRight (http://localhost:3000/static/js/main.chunk.js:14193:3)
    at header
    at Header (http://localhost:3000/static/js/main.chunk.js:14357:3)
    at DigitalHeader (http://localhost:3000/static/js/main.chunk.js:17325:3)
    at main
    at div
    at Provider (http://localhost:3000/static/js/vendors~main.chunk.js:49547:20)